jo-rdan/Free-Mentor
# #167636061 User can review a mentor

#### What does this PR do?
It creates a user interface popup menu to allow a user to rate a mentor
#### Description of Task to be completed?
- create popup menu 
- add icon for rating
#### How should this be manually tested?
- Clone this repository to your computer
- Open the html file named **specific-mentor** then click on the flag icon and see what happens. 
#### Any background context you want to provide?
- This is only the design implemented no core functionalities
#### What are the relevant pivotal tracker stories?
[#167636061](https://www.pivotaltracker.com/story/show/167636061)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/51836855/63048180-db912e80-be8a-11e9-9676-7a9f0d726c9b.png)
